Command/RunCommand.php

```diff
@@ -69,7 +67,7 @@ class RunCommand extends Command
 
     public function __construct(ManagerRegistry $managerRegistry, JobManager $jobManager, EventDispatcherInterface $dispatcher, array $queueOptionsDefault, array $queueOptions)
     {
-        parent::__construct();
+        parent::__construct('jms-job-queue:run');
```

Comment:
I had to move the name to the constructor because the constructor calls `configure` at the very end. And if you then call `setName` after the constructor already ran you can't extend commands anymore. This normally is not a problem, but Indiana extends the `RunCommand` (`app:jms:run`) because we have to add an additional parameter ("hostname"); and this `setName` made our command unavailable as the original one took precendence.
And there is also some extra logic regarding aliases in the constructor that is not executed in this case, but we don't need that

Reply:
Oh damn, good point.